frontend/src/index.css

```diff
@@ -0,0 +1,14 @@
+body {
+  margin: 0;
+  font-family: "Lato", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: rgba(233, 235, 242, 0.2);
+
+  --blue-grey: #999fa7;
```

Comment:
I was going to use zeplins token names but only ended up using one css variable heh. This is hopes and dreams of organized css.